src/algebra/pointwise.lean

```diff
@@ -238,6 +238,16 @@ Union_image_left _
 lemma Union_mul_right_image [has_mul α] : (⋃ a ∈ t, (λ x, x * a) '' s) = s * t :=
 Union_image_right _
 
+@[to_additive]
+lemma Union_mul_right {α ι : Type*} [has_mul α] (s : ι → set α) (t : set α) :
+  (⋃ i, s i) * t = ⋃ i, (s i * t) :=
+image2_Union_left _ _ _
+
+@[to_additive]
+lemma Union_mul_left {α ι : Type*} [has_mul α] (s : ι → set α) (t : set α) :
+  t * (⋃ i, s i) = ⋃ i, (t * s i) :=
+image2_Union_right _ _ _
```

Comment:
To match the lowercase versions, I'd suggest
```suggestion
@[to_additive]
lemma Union_mul {α ι : Type*} [has_mul α] (s : ι → set α) (t : set α) :
  (⋃ i, s i) * t = ⋃ i, (s i * t) :=
image2_Union_left _ _ _

@[to_additive]
lemma mul_Union {α ι : Type*} [has_mul α] (s : ι → set α) (t : set α) :
  t * (⋃ i, s i) = ⋃ i, (t * s i) :=
image2_Union_right _ _ _
```

Reply:
I guess the same applies for `mul_supr` and `supr_mul` as the name of the submodule lemmas.